src/main/webapp/partials/instructorHelpCourses.jsp

```diff
@@ -821,6 +833,282 @@
+                   data-toggle="tooltip"
+                   data-placement="top">
+                  <span class="glyphicon glyphicon-remove"></span>
+                  <strong>Delete All</strong>
+                </a>
+                <a class="btn btn-default btn-xs pull-right pull-down"
+                   data-toggle="tooltip"
+                   data-placement="top">
+                  <span class="glyphicon glyphicon-ok"></span>
+                  <strong>Restore All</strong>
+                </a>
+                <strong>
+                  Recycle Bin
+                </strong>
+              </div>
+              <div class="panel-collapse">
+                <div class="panel-body padding-0">
+                  <table class="table table-bordered table-striped margin-0">
+                    <thead class="background-color-medium-gray text-color-gray font-weight-normal">
+                    <tr>
+                      <th onclick="toggleSort(this);" class="button-sort-none">
+                        Course ID<span class="icon-sort unsorted"></span>
+                      </th>
+                      <th onclick="toggleSort(this);" class="button-sort-none">
+                        Course Name<span class="icon-sort unsorted"></span>
+                      </th>
+                      <th onclick="toggleSort(this);" class="button-sort-none">
+                        Creation Date<span class="icon-sort unsorted"></span>
+                      </th>
+                      <th onclick="toggleSort(this);" class="button-sort-none">
+                        Deletion Date<span class="icon-sort unsorted"></span>
+                      </th>
+                      <th class="align-center no-print">Action(s)</th>
+                    </tr>
+                    </thead>
+                    <tr>
+                      <td>AI532</td>
+                      <td>Artificial Intelligence</td>
+                      <td>20 Jan 2018</td>
+                      <td>10 Mar 2018</td>
+                      <td class="align-center no-print">
+                        <button href="#" class="btn btn-default btn-xs" type="button">Restore</button>
+                        <button href="#" class="btn btn-default btn-xs" type="button" style="color: #E74C3C">Delete Permanently</button>
+                      </td>
+                    </tr>
+                  </table>
+                </div>
+              </div>
+            </div>
+          </div>
+          <p>
+            After restoring the course, all the information relevant to the course (e.g. instructors, sessions, students) will also be restored.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="panel panel-default" id="course-delete">
+      <div class="panel-heading cursor-pointer" data-toggle="collapse" data-target="#course-delete-body">
+        <h3 class="panel-title">How do I pernamently delete a course?</h3>
```

Comment:
There's a typo here: it's "permanently", not "pernamently" :P

Reply:
Sorry my bad :(